packages/react-components/src/components/search-bar/index.tsx

```diff
@@ -25,6 +25,36 @@ type Props = {
     clear?: boolean
 }
 
+const Container = styled.div`
```

Comment:
I upgraded something and now you have to declare things before they are used, which sounds like a good idea